test/factories/comment.rb

```diff
@@ -1,5 +1,6 @@
 FactoryBot.define do
 	factory :comment do 
+		subject
```

Comment:
Indentation is off here

Reply:
weird, I thought Rubo would pick these up!?